studio/components/layouts/ProjectLayout/ProjectPausedState.tsx

```diff
@@ -53,7 +58,7 @@ const ProjectPausedState = ({ product }: ProjectPausedStateProps) => {
     }
 
     await post(`${API_URL}/projects/${project.ref}/restore`, { kps_enabled: kpsEnabled })
-    app.onProjectUpdated({ ...project, status: PROJECT_STATUS.RESTORING })
+    await invalidateOrganizationsQuery(queryClient)
```

Comment:
same comment as per `ComputeInstanceSizePanel`

Reply:
this one is just completely wrong anyway 🥲